docs/UsingCppInterOp.rst

```diff
@@ -67,33 +67,204 @@ that it was designed for.
 
 Further Enhancing the Dynamic/Automatic bindings in CPPYY
 =========================================================
+
+CppInterOp ties into the following major enhancements in cppyy:
+
+-  **Reducing dependencies** of cppyy to minimize code bloat and make it
+   faster.
+
+-  A **cppyy-Style Library** (i.e., CppInterOp) that enables interoperability
+   with C++ code, bringing the speed and efficiency of C++ to simpler, more
+   interactive languages like Python.
+
+**Reducing Dependencies**
+
+Recent work done on cppyy has been focused on removing unnecessary
+dependencies on domain-specific infrastructure (e.g., the ROOT
+framework). The idea was to convert the cppyy-backend to use Cling
+directly (instead of ROOT meta), and then use it in cppyy.
```

Comment:
link to root meta for the interested reader

Reply:
linked to https://github.com/root-project/root/tree/master/core/meta